setup.py

```diff
@@ -17,6 +17,7 @@
     package_data={
         "web_poet": ["py.typed"],
     },
+    entry_points={"pytest11": ["web-poet = web_poet.testing.pytest"]},
```

Comment:
I've found that the coverage report gets broken by this line, not sure why yet.

Reply:
https://stackoverflow.com/questions/62221654/how-to-get-coverage-reporting-when-testing-a-pytest-plugin

https://pytest-cov.readthedocs.io/en/latest/plugins.html

Reply:
I've used the SO idea of running `coverage` directly because the idea from the second link is both complicated and leads to several additional problems, some of which have open issues in pytest-cov. The benefits of pytest-cov are listed at https://pytest-cov.readthedocs.io/en/latest/readme.html , hopefully we can live withouth them.